src/flask_container_scaffold/base.py

```diff
@@ -1,3 +1,5 @@
+import warnings
```

Comment:
(question, non-blocking) This is interesting, I wasn't familiar with the 'warnings' filter at all. Is this because we don't necessarily have a logger available in this case?